fix: Link dwarf in fuzzing builds too

Sarkoxed opened the pull request:
This pr adds a link to dwarf library in fuzzing setup whenever it is present on a system

Also added the required packages for fuzzing container build

ludamad reviewed (commented):
approved but you should remove CHECK_CIRCUIT_STACKTRACES as this is now implied by ENABLE_STACK_TRACES

Sarkoxed committed:
fix: Link dwarf in fuzzing builds too

This pr adds a link to dwarf library in fuzzing setup whenever it is present on a system

Also added the required packages for fuzzing container build

Co-authored-by: Tom French <15848336+TomAFrench@users.noreply.github.com>
